src/XenobladeChroniclesX/Mods/BattleDamageModGround2/patch_dmg.asm

```diff
@@ -0,0 +1,22 @@
+[XCX_DAMAGES] ####################################################################################################################################
```

Comment:
Can we remove the `###########`'s? I'm not sure if these come from another pack, but unnecessary comments are not really that helpful.

Reply:
I'm even more late than you haha, sorry. I'll do that, just trying to see if I can do it on GitHub or I need to commit with the CLI.

Reply:
Added a commit.